### PR TITLE
Merge development with master

### PR DESCRIPTION
**Pull request by Nicklason**
- !deposit, !withdraw, !cart, !clearcart and !checkout commands
- Fixed problem with enabling / disabling autobump

### CODE REVIEW

File: src/app/handler/cart.js [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+        } else {
+            message = pluralize(name, amount, true) + (amount > 1 ? ' have' : ' has') + ' been removed from ' + whose + ' side of the cart';
+        }
+    }
+
+    getCart(steamID).remove(sku, amount, side);
+
+    if (getCart(steamID).isEmpty()) {
+        deleteCart(steamID);
+        return { cart: undefined, message };
```
Comment (ZeusJunior) [inline src/app/handler/cart.js]: `{ cart: undefined, message };` is this correct?

File: src/app/handler/cart.js [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+exports.stringify = function (steamid) {
+    const cart = getCart(steamid);
+
+    if (!cart) {
+        return 'Your cart is empty';
+    }
+
+    if (cart.isEmpty()) {
+        return 'Your cart is empty';
+    }
```
Comment (ZeusJunior) [inline src/app/handler/cart.js]: Double?

Comment (Nicklason) [reply src/app/handler/cart.js]: If there is no cart, or it is empty, then it is empty

File: src/app/handler/cart.js (hunk 1 above)
Comment (Nicklason) [reply src/app/handler/cart.js]: It is correct yes

File: src/app/handler/cart.js (hunk 2 above)
Comment (ZeusJunior) [reply src/app/handler/cart.js]: Why not combine them with ||

Comment (ZeusJunior) [reply src/app/handler/cart.js]: Wait, nope. Cant do that if cart doesn't exist B)

Comment (Nicklason) [reply src/app/handler/cart.js]: You can, I'll make a commit

File: src/app/handler/cart.js (hunk 1 above)
Comment (ZeusJunior) [reply src/app/handler/cart.js]: Didn't know that was possible, cool

**Pull request merged**